lib/createMd.js

```diff
@@ -40,7 +40,7 @@ function showState(state) {
 
     line.push(state.indicator ? 'X' : ' ');
     //line.push(state.noSubscribe ? 'NS' : '');
-    line.push(state.role ? '`' + state.role.replace(/\|/g, '\|') + '`' : ''); //escape | character inside table cell!
+    line.push(state.role ? '`' + state.role.replace(/[|`]/g, '\|') + '`' : ''); //escape | character inside table cell!
```

Comment:
Sorry, this will replace backtick with backslash-pipe.

Reply:
ups ;-)